.github/workflows/build_conda_linux.yml

```diff
@@ -174,17 +174,16 @@ jobs:
             ${CONDA_RUN} python "${{ inputs.repository }}/${SMOKE_TEST_SCRIPT}"
           fi
       - name: Upload package to conda
-        if: ${{ inputs.trigger-event == 'push' }}
+        # if: ${{ inputs.trigger-event == 'push' }}
```

Comment:
Will undo this change (and similar one below) before merging. Keeping this around until then so that tests on this PR actually do the upload.